### PR TITLE
test: added Cypress testing suite

### PR DESCRIPTION
**Pull request by gloriliale**
Added the Cypress testing suite for integrated testing needs. Alongside this, created a command in the package.json for easy access to Cypress, and added a line explaining how to use this command in the README.

### CODE REVIEW

File: cypress.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (nkrantz) [inline cypress.json:1]: Not sure whether we want/need any special config settings, but [here are the options](https://docs.cypress.io/guides/references/configuration.html#Options) just for reference

File: package.json [hunk 2]
```diff
@@ -9,11 +9,13 @@
     "lint": "eslint --ext js ./",
     "lint:fix": "npm run lint -- --fix",
     "prettier": "prettier --list-different './**/*.{js,jsx}'",
-    "prettier-clean": "prettier --write './**/*.{js,jsx}'"
+    "prettier-clean": "prettier --write './**/*.{js,jsx}'",
+    "cypress": "cypress open"
```
Comment (nkrantz) [inline package.json:13]: Super useful to have the script written and also in the README.md, thanks!!

**Review by nkrantz — COMMENTED** [review]
Looks great! Thanks for adding cypress. Not sure whether this branch also has to be updated with eslint/prettier before being merged to main... I'll take a look at what that process is

File: cypress.json (hunk 1 above)
Comment (gloriliale) [reply cypress.json:1]: Great suggestion, thanks for the link Nora!

File: package.json (hunk 2 above)
Comment (gloriliale) [reply package.json:13]: Hooray for less typing!

**Review by SiTaggart — APPROVED** [review]

**Pull request merged**